cli/command/image/load.go

```diff
@@ -49,6 +50,9 @@ func runLoad(dockerCli *command.DockerCli, opts loadOptions) error {
 		defer file.Close()
 		input = file
 	}
+	if opts.input == "" && dockerCli.In().IsTerminal() {
+		return fmt.Errorf("a tar file is not given (e.g. docker load < image.tar)")
```

Comment:
And maybe a comment above the `if` so other people know what this is for specifically.